test/integration/sds_dynamic_integration_test.cc

```diff
@@ -179,29 +223,71 @@ class SdsDynamicDownstreamIntegrationTest : public SdsDynamicIntegrationBaseTest
     client_ssl_ctx_ = createClientSslTransportSocketFactory({}, context_manager_, *api_);
   }
 
+  void configToUseSds(
+      envoy::extensions::transport_sockets::tls::v3::CommonTlsContext& common_tls_context) {
+    common_tls_context.add_alpn_protocols(downstream_protocol_ == Http::CodecClient::Type::HTTP1
+                                              ? Http::Utility::AlpnNames::get().Http11
+                                              : Http::Utility::AlpnNames::get().Http3);
```

Comment:
Should this use `test_quic` instead? I think this logic just matches the logic above that uses `test_quic`? Not clear without that why we do this remapping

Reply:
Yes, test_quic_ and downstream_protocol_ are consistent. Switched to use test_quic_ as other places in this test do.